packages/gatsby-graphql-source-toolkit/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "gatsby-graphql-source-toolkit",
+  "version": "1.0.0",
+  "description": "The toolkit designed to simplify data sourcing from a remote GraphQL API into Gatsby.",
+  "author": "Vladimir Razuvaev <vladimir@gatsbyjs.com>",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "babel src --out-dir lib --ignore \"**/__tests__\" --extensions \".ts,.js,.tsx\"",
+    "prepare": "cross-env NODE_ENV=production npm run build && npm run typegen",
+    "typegen": "rimraf \"lib/**/*.d.ts\" && tsc --emitDeclarationOnly --declaration --declarationDir lib/",
+    "typecheck": "tsc --noEmit --diagnostics",
+    "watch": "babel -w src --out-dir lib --ignore \"**/__tests__\"  --extensions \".ts,.js,.tsx\"",
+    "lint:code": "eslint --cache --ext .js,.jsx,.ts,.tsx src"
+  },
+  "files": [
+    "lib/**"
+  ],
+  "keywords": [
+    "gatsby",
+    "graphql"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gatsbyjs/gatsby.git",
+    "directory": "packages/gatsby-graphql-source-toolkit"
+  },
+  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-graphql-source-toolkit#readme",
+  "dependencies": {
+    "@types/node-fetch": "^2.5.7",
```

Comment:
Does this need to be dependency? I feel that this should be devDependency

Reply:
Good catch, will move it to `devDependencies`